src/components/ActivitiesBarChart/index.js

```diff
@@ -0,0 +1,123 @@
+import React from 'react'
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  ResponsiveContainer,
+} from 'recharts'
+
+import { Wrapper, Title } from './styles'
+
+import PropTypes from 'prop-types'
+
+export const ActivitiesBarChart = ({ data }) => {
+  return (
+    <Wrapper>
+      <Title>Gráfico de barras das suas atividades complementares</Title>
+      <ResponsiveContainer minWidth={380} width="100%" height={400}>
+        <BarChart
+          wrapperStyle={{ paddingLeft: 0 }}
+          data={data}
+          margin={{
+            top: 0,
+            right: 0,
+            left: 0,
+            bottom: 0,
+          }}
+        >
+          <CartesianGrid strokeDasharray="3 3" />
+          <XAxis dataKey="name" />
+          <YAxis dataKey="top" width={30} />
+          <Tooltip />
+          <Legend />
+          <Bar
+            legendType="rect"
+            dataKey="Creditos totais"
+            BarSize={42}
+            fill="#975858"
+          />
+          <Bar
+            legendType="rect"
+            dataKey="Max creditos"
+            BarSize={42}
+            fill="#343B4570"
+          />
+        </BarChart>
+      </ResponsiveContainer>
+    </Wrapper>
+  )
+}
+
+ActivitiesBarChart.propTypes = {
+  data: PropTypes.shape([
+    {
+      name: PropTypes.string,
+      'Creditos totais': PropTypes.string,
+      'Max creditos': PropTypes.string,
+      top: PropTypes.number,
+    },
+  ]),
+}
+
+ActivitiesBarChart.defaultProps = {
```

Comment:
Tenho a impressão que isso é um pouco perigoso. Acredito que talvez esse `defaultProps` possa levar a alguns bugs de renderização onde o props não é passado para o componente, é renderizado o componente com `defaultProps` e achamos que está tudo certo. Novamente, é só uma impressão minha e não sei o real impacto disso.

Reply:
Essa parte dos props padrão eu deixei mesmo por causa da PR, pra conseguir visualizar, aí quando a gente for desenvolver de fato eu vou tirar

Reply:
Perfeito então!